packages/tooling/cypress/support/e2e.js

```diff
@@ -18,8 +18,9 @@ Cypress.on('uncaught:exception', (err) => {
   // Cypress and React Hydrating the document don't get along
   // for some unknown reason. Hopefully, we figure out why eventually
   // so we can remove this.
+
   if (
-    /hydrat/i.test(err.message) ||
+    /ydrat/i.test(err.message) ||
```

Comment:
Is this an accidental deletion?

Also it looks like a copy-paste from a GitHub issue https://github.com/cypress-io/cypress/issues/27204#issuecomment-1894217901. I think we should have an issue in Volto's issue tracker to track it and remove it when it is fixed. That will also allow you to spread the maintenance workload and relieve your brain of being the sole source.

Reply:
Not a typo, is fine. Sometimes the error message has a Capital letter (because it's the start of a sentence) and in other occasions, it does not. The error is still present in Cypress 14 unfortunately :(

Anyways, it's a minor thing.